apps/blog/content/posts/what-is-memebattle.en.mdx

```diff
@@ -7,8 +7,106 @@ tags: ['MemeBattle']
+The hackathon was about entertainment and fun. We created an application (web and Android) with an infinity battle of memes. Every few seconds users saw a pair of memes and voted for one of them.
+Players who rated the meme with a majority vote received the internal currency – MemeCoin.
+We won that hackathon and received investments for development.
+
+<details className="my-6">
+  <summary>First application version screenshots</summary>
+  <img src="/content-images/what-is-memebattle/memebattle_v1.png" alt="MemeBattle v1. Screenshow of web application"/>
+  <img src="/content-images/what-is-memebattle/memebattle-playmarket.png" alt="MemeBattle v1. Screenshot of application page on google play"/>
+</details>
+
+After that we have participated in hackathons several years. Local hackathons in Siberia (Tomsk, Kemerovo, Novosibirsk) and grand ones in Moscow – it was our hobby.
+Meanwhile, we periodically returned to the original idea, but didn't create real product 😢
+
+<InstagramPost publicationId="BaRdqExl35J" description="A post shared by Themezv (@artem_zverev.9)" />
+
+In parallel with hackathons we have done some education projects. For example, in union with other students, organized IT community AVTF – few meetups by different streams (web development, mobile development, databases, information security, data science). Just so you know, on the first meetup were more than 150 people!
+
+<Image src='/content-images/what-is-memebattle/ITC2.jpg' alt="Photo from first meetup of IT community AVTF" width={1920} height={925} />
+
+We often discussed about MemeBattle and told to friend. But we could never answer the question "What is MemeBattle". Usually we respond like:
+> MemeBattle – is not a dot on a map. MemeBattle – is people.
+
+<InstagramPost publicationId="BuPCRKrlI4C" description="A post shared by MemeBattle (@memebattle.world)" />
+
+Looking back, I can answer:
+><b>MemeBattle</b> – small community around few technical projects, where we studied new technologies and approaches. <br /> Also this is the team with which we participated in hackathons and other activities.
+
+## MemeBattle today
+
+The initial concept of an application centered around battle of memes is a thing of the past.
+
+At the moment MemeBattle is primarily an <b>open community</b> united by common projects.
+We are open to new ideas and projects, also we invite everyone to join us.
+
+There are few projects in development by MemeBattle today.
+
+### Ligretto
+
+<b>Ligretto</b> – dynamic online multiplayer game, where needed to put cards as soon as possible and outperform opponents with reaction and dexterity.
+
+url: [https://ligretto.app/](https://ligretto.app/)
+
+Play with friends anytime – one round takes only few minutes. Our colleagues spend a free time between meetings sometimes)
+
+### GameHub
+
+<b>GameHub</b> – online games service. Unfortunately there is only one game so far. But in the ideal future it will be the portal of online games in one ecosystem.
```

Comment:
GameHub is an online gaming service. Unfortunately, there is only one game so far. But in the ideal future, it will be a portal for multiple online games in one ecosystem.